yt_dlp/extractor/mojevideo.py

```diff
@@ -0,0 +1,43 @@
+import re
+from .common import InfoExtractor
+from ..utils import ExtractorError
+
+
+class MojevideoIE(InfoExtractor):
+    _VALID_URL = r'https://www\.mojevideo\.sk/video/(?P<id>[0-9]+)'
+
+    _TESTS = [{
+        'url': 'https://www.mojevideo.sk/video/3d17c/chlapci_dobetonovali_sme_mame_hotovo.html',
+        'md5': '384a4628bd2bbd261c5206cf77c38c17',
+        'info_dict': {
+            'id': '250236',
+            'ext': 'mp4',
+            'title': 'Chlapci dobetónovali sme, máme hotovo!',
+            'description': 'Celodenná práca bola za pár sekúnd fuč. Betón stiekol k susedovi, kam aj zrútil celý plot, ktorý polámal aj tuje. Chlapom zostali iba oči pre plač.'
+        }
+    }]
+
+    def _real_extract(self, url):
+        webpage = self._download_webpage(url, 1)
+
+        video_id = re.search(r'vId=(\d+)', webpage).group(1)
+        video_expiration = re.search(r"vEx='(\d+)'", webpage).group(1)
+        video_hash = re.search(r'vHash=\[([^\]]+)', webpage).group(1).split(",")[0].replace("'", "")
+        video_title = re.search(r'<h1>(.*?)</h1>', webpage).group(1)
+        video_description = re.search(r'<div id="video-comment">.*?<p>(.*?)</p>', webpage, re.DOTALL).group(1)
```

Comment:
avoid using `.*?` in regex, esp when the search pattern is so broad. this can cause catastrophic backtracking. for this we should just use `_og_search_description()`. but again, we should inline it in the return, so let's remove
```suggestion
```